docker-entrypoint.sh

```diff
@@ -9,7 +67,17 @@ cronSchedule=${cronSchedule%\"}
 cronSchedule=${cronSchedule#\"}
 
 # Create crontab definition
-echo "$cronSchedule . /container_env.sh; /usr/local/bin/backup.sh >> /var/log/cron.log 2>&1" > /etc/cron.d/crontab.conf
+if [[ $SMTP_HOST ]];
+then
+    echo "Cron e-mail reporting activated. '${SMTP_HOST}'"
+    # (GH-39) Add (-a) paramter to append the log file, otherwise the log file is truncated every run.
+    # echo "$cronSchedule . /container_env.sh; /usr/local/bin/backup.sh 2>&1 | tee -a /var/log/cron.log | mail -s 'SQL Server Backup Result' $MAIL_TO 2>&1 | tee -a /var/log/cron.log" > /etc/cron.d/crontab.conf
+    echo "$cronSchedule . /container_env.sh; /usr/local/bin/backup.sh 2>&1 | tee /var/log/cron.log | mail -s 'SQL Server Backup Result' $MAIL_TO 2>&1 | tee /var/log/cron.log" > /etc/cron.d/crontab.conf
```

Comment:
I think the final ` > /etc/cron.d/crontab.conf` is not needed. That's what `tee` does already. It writes to stdout and forwards to the file.

Reply:
IMHO this writes the cron configuration to the config file.